### ISSUE TITLE
no mLOY result but normal mLOX and mCA of male in autosomes

### ISSUE THREAD

**Issue by caragraduate** [freeseek/mocha#39]
Hi,

Thank you for this wonderful tools! As I am applying this tool on my project and I have a question that cannot find the answer by myself and hope to get some clues from you.

I have successfully received the results from mocha and I would like to get the mLOY and mLOX. I can get numbers of mLOX from the results but never get an any mLOY samples (I have ~half of the samples are male in the original genotype data). Are there any potential reasons you this may cause this happen or do you have any suggestions how can I trouble shoot this?

Thank you for any of your incoming suggestions!

**Comment by freeseek** [comment]
Because mLOY is called through the PAR1 region, it is called as a chromosome X imbalance event. You can see how to generate mLOY phenotypes [here](https://github.com/freeseek/mocha#mosaic-phenotypes)

**Comment by caragraduate** [comment]
> Because mLOY is called through the PAR1 region, it is called as a chromosome X imbalance event. You can see how to generate mLOY phenotypes [here](https://github.com/freeseek/mocha#mosaic-phenotypes)

Thank you for your timely reply! Sry for the confusion and please let me make myself more clear. Yes, I actually already called mLOY and mLOX used the same command on the link you shared. But for all of my datasets, there are no mLOY sample but do have many mLOX samples. Is that possible? I am expecting more mLOY actually but I have no clue which steps or any other reasons could cause this issue. I did not have any other errors in the previous steps. Would be much appreciated if you could give some insights on this.

Also, one side question I am not sure if it is related to the current issue (probably not), so in the tutorial page of mocha, you are only using autosome and chrX, we never concerns about chrY, is that correct? Is there any reason we exclude the chrY during the analysis?

Thank you for your time and help!

**Comment by freeseek** [comment]
As I said, only PAR1 probes are used for the Y chromosome, so mLOY will be labeled as a chromosome X event in the output callset

**Comment by caragraduate** [comment]
> As I said, only PAR1 probes are used for the Y chromosome, so mLOY will be labeled as a chromosome X event in the output callset

Thank you! So are you saying it is normal to see my $pfx.Y_loss.lines file has no sample inside? And I do have few samples in the $pfx.X_loss.lines. I understand mLOY will be labeled as a chrX event, but if there is no samples in the $pfx.Y_loss.lines, how should we identify mLOY event then?

**Comment by caragraduate** [comment]
> You certainly could have a dataset with no mLOY detected, for example, if all samples were females

Yes you are totally correct, but like I said, I have ~half of the samples are male in our data, but still no mLOY detected. But anyway I just observed that this could because that our genotype data is missing coverage for both of the PAR1 and PAR2. This might explain why we only got mLOX sample and did not have any mLOY samples. Thank you for your time again!
